binr/rax2/rax2.c

```diff
@@ -75,6 +75,24 @@ static void print_ascii_table() {
 	printf("%s", ret_ascii_table());
 }
 
+static int escape_to_dec(int i, char *str) {
+	switch (str[i]) {
```

Comment:
i think the index should be handled by the caller, not the function otherweise you have a boundary issue in here. so just switch(ch) {